Makefile

```diff
@@ -42,45 +60,25 @@ translations: $(po_files)  ## Update translation files
 	@echo 'where `fr` is the language code.'
 	@echo
 
-.PHONY: static
-static: $(css_files)  ## Compile the less->css
-$(module_root)/public/%.css: $(module_root)/public/%
-	@echo "$< -> $@"
-	node_modules/less/bin/lessc $< $@
-
-.PHONY: requirements
-requirements:  # Install required packages
-	pip install tox
-	npm install
+_NAME=stvstnfrd/image-modal:latest
+_VOLUME=-v '$(PWD):/root/xblock'
+_PORT=
+runserver: _PORT = -p 8000:8000
+docker_test: _VOLUME = -v '$(PWD)/reports:/root/xblock/reports'
+docker_run=docker run $(_PORT) $(_VOLUME) --rm -it
+docker_make=$(docker_run) --entrypoint make $(_NAME)
+docker_make_args=language=$(language) -C /root/xblock
+docker_make_more=$(docker_make) $(docker_make_args)
 
-.PHONY: run
-# This target is intentionally hidden from the help menu,
-# as it shouldn't be invoked directly.
-run:  # Run the workbench server w/ this XBlock installed
-	cd $(sdk_root) && pip install -r requirements/base.txt
-	pip install -e .
-	cd $(sdk_root) && python manage.py migrate
-	cd $(sdk_root) && python manage.py runserver 0.0.0.0:8000
-
-.PHONY: test
-test: requirements  ## Run all quality checks and unit tests
-	tox
-
-.PHONY: vagrant_clean
-vagrant_clean: vagrant_halt clean  ## Remove build artifacts (and destroy vagrant VM)
-	vagrant destroy -f
-
-.PHONY: vagrant_halt
-vagrant_halt:  ## Stop running vagrant VM vagrant halt
-	vagrant halt
-
-define run-in-vagrant
-	vagrant up
-	vagrant ssh -c ". /home/vagrant/venv/bin/activate && $(MAKE) language=$(language) -C /home/vagrant/xblock/ $(patsubst vagrant_%, %, $@)"
+.PHONY: build_docker
+build_docker:
```

Comment:
Renamed and removed comment.
Removing the comment means this won't show up in the help text (intentionally)
since it's now run automatically everywhere it's needed as a dependency.

Reply:
good idea